assets/js/app.js

```diff
@@ -53,6 +53,38 @@ let liveSocket = new LiveSocket('/live', Socket, {
   },
 });
 
+window.addEventListener('phx:historical-events-load', (e) => {
+  const tracesListDiv = document.querySelector(
+    `#${e.detail.trace_list_dom_id}`
+  );
+  let separator = document.querySelector('#separator');
+
+  if (separator) {
+    tracesListDiv.removeChild(separator);
+  } else {
+    separator = document.createElement('div');
+    separator.id = 'separator';
+    separator.innerHTML = `
+      <div class="border-b h-0 border-primary-100"></div>
+      <div class="px-6 py-1 text-primary font-normal text-center text-xs">Historical events</div>
```

Comment:
why don't you add `border-y` to the middle `div`? It should probably do the trick